addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -1,41 +1,328 @@
+            ) / (product_tot_qty_available + qty)
+        std_price_update[move.company_id.id, move.product_id.id] = new_std_price
+
+
+def _create_in_svl(env, move, std_price_update):
+    move = move.with_context(force_company=move.company_id.id)
+    valued_move_lines = move._get_in_move_lines()
+    valued_quantity = 0.0
+    for valued_move_line in valued_move_lines:
+        valued_quantity += valued_move_line.product_uom_id._compute_quantity(valued_move_line.qty_done,
+                                                                             move.product_id.uom_id)
+    _change_standard_price(env, move, std_price_update)  # Create SVL for manual updated standard price
+    unit_cost = abs(_get_price_unit(move, std_price_update))  # May be negative (i.e. decrease an out move).
+    if move.product_id.cost_method == 'standard':
+        unit_cost = std_price_update.get((move.company_id.id, move.product_id.id), move.product_id.standard_price)
+    vals = move.product_id._prepare_in_svl_vals(valued_quantity, unit_cost)
+    vals.update(move._prepare_common_svl_vals())
+    vals.update({
+        'create_date': move.date,
+        'create_uid': move.create_uid.id,
+        'write_date': move.date,
+        'write_uid': move.create_uid.id,
+    })
+    _create_stock_valuation_layer(env, [vals])
+
+
+def _create_out_svl(env, move, std_price_update):
+    move = move.with_context(force_company=move.company_id.id)
+    valued_move_lines = move._get_out_move_lines()
+    valued_quantity = 0.0
+    for valued_move_line in valued_move_lines:
+        valued_quantity += valued_move_line.product_uom_id._compute_quantity(valued_move_line.qty_done,
+                                                                             move.product_id.uom_id)
+    if float_is_zero(valued_quantity, precision_rounding=move.product_id.uom_id.rounding):
+        return
+    _change_standard_price(env, move, std_price_update)
+    vals = move.product_id._prepare_out_svl_vals(valued_quantity, move.company_id)
+    vals.update(move._prepare_common_svl_vals())
+    vals.update({
+        'create_date': move.date,
+        'create_uid': move.create_uid.id,
+        'write_date': move.date,
+        'write_uid': move.create_uid.id,
+    })
+    _create_stock_valuation_layer(env, [vals])
+
+
+def _create_dropshipped_svl(env, move, std_price_update):
+    svl_vals_list = []
+    move = move.with_context(force_company=move.company_id.id)
+    valued_move_lines = move.move_line_ids
+    valued_quantity = 0.0
+    for valued_move_line in valued_move_lines:
+        valued_quantity += valued_move_line.product_uom_id._compute_quantity(valued_move_line.qty_done,
+                                                                             move.product_id.uom_id)
+    quantity = valued_quantity
+
+    _change_standard_price(env, move, std_price_update)
+    unit_cost = _get_price_unit(move, std_price_update)
+    company = move.product.env.company
```

Comment:
I'm getting this error during migrations
```
2020-09-18 09:02:01,907 1 ERROR prod OpenUpgrade: stock_account: error in migration script stock_account/migrations/13.0.1.1/post-migration.py: 'stock.move' object has no attribute 'product' 
2020-09-18 09:02:01,907 1 ERROR prod OpenUpgrade: 'stock.move' object has no attribute 'product' 
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/openupgradelib/openupgrade.py", line 1795, in wrapped_function
    if use_env2 else cr, version)
  File "/opt/odoo/auto/addons/stock_account/migrations/13.0.1.1/post-migration.py", line 325, in migrate
    generate_stock_valuation_layer(env)
  File "/opt/odoo/auto/addons/stock_account/migrations/13.0.1.1/post-migration.py", line 32, in generate_stock_valuation_layer
    _create_dropshipped_svl(env, move, std_price_update)
  File "/opt/odoo/auto/addons/stock_account/migrations/13.0.1.1/post-migration.py", line 121, in _create_dropshipped_svl
    company = move.product.env.company
AttributeError: 'stock.move' object has no attribute 'product'
2020-09-18 09:02:01,910 1 ERROR prod odoo.modules.loading: Error executing post migration script for module stock_account
```

Reply:
try using https://github.com/ForgeFlow/OpenUpgrade/pull/17

Reply:
Thanks for your response @HviorForgeFlow.
I fixed it. Please check again.